src/util/stringFormatting.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Function that accepts string and replaces underscores with spaces
+ *
```

Comment:
no need to have a space after the function description

Reply:
done